STATUS: INLINE SUCCESSFUL SETUPTEARDOWN
MODIFIED: io.jboot.test.cache.redis.RedisCacheTester.testGet
MODIFIED FILE: /home/lichengpeng/test2/automated_script/yangfuhai/jboot/src/test/java/io/jboot/test/cache/redis/RedisCacheTester.java
CLEANER: io.jboot.test.cache.ehcache.EhCacheTester.testPut
CLEANER FILE: /home/lichengpeng/test2/automated_script/yangfuhai/jboot/src/test/java/io/jboot/test/cache/ehcache/EhCacheTester.java
POLLUTER: N/A
POLLUTER FILE: N/A
ORIGINAL CLEANER SIZE: 7
NEW CLEANER SIZE: 1
ELAPSED TIME: 35.437999963760376
==========================
@@ -35,0 +35,3 @@
+ {
+     JbootApplication.setBootArg("jboot.cache.type", "ehcache");
+ }
